plutus-core/plutus-core/src/PlutusCore/FilePaths.hs

```diff
@@ -0,0 +1,24 @@
+-- | Various file paths used in plutus-core, mostly related to the cost model.
+
+module PlutusCore.FilePaths
```

Comment:
maybe `DataFilePaths`? This sounds like it's going to contain functions for working with file paths or something

Reply:
> Seems fine. I have no idea what will actually happen in a cross setting with this.

Me neither, but let's live a little and do it regardless.